poc/guests/sum-balance-percent/src/main.rs

```diff
@@ -5,9 +5,9 @@ static GLOBAL: polkavm_derive::LeakingAllocator = polkavm_derive::LeakingAllocat
 use alloc::vec::Vec;
 #[xcq_api::program]
 mod sum_balance {
-    #[xcq::call_def]
+    #[xcq::call_def(extension_id = 10588899351449456541u64, call_index = 1)]
```

Comment:
maybe is better to specify extension_id by hex? e.g. `0x12345u64` so that it is shorter